src/Sensor_TEST.cc

```diff
@@ -247,7 +247,7 @@ TEST(DOMSensor, Type)
     sdf::SensorType::CONTACT,
     sdf::SensorType::DEPTH_CAMERA,
     sdf::SensorType::FORCE_TORQUE,
-    sdf::SensorType::GPS,
+    sdf::SensorType::NAVSAT,
```

Comment:
Should we leave `sdf::SensorType::GPS` in this test since we support both options?

Reply:
Since the enum set by `sdf::Sensor::Load` will be `NAVSAT` for either `gps` or `navsat` strings, I don't think we can test this.